rollup.config.js

```diff
@@ -1,13 +1,19 @@
+// Rollup plugins 
 import babel from 'rollup-plugin-babel';
 
 export default {
   input: 'src/index.js',
-  output: {
+  output: [{
     file: 'lib/index.js',
     name: 'Syft',
     format: 'iife',
     sourceMap: true
   },
+  {
+    file: 'lib/index.esm.js',
+    format: 'esm'
+  } 
```

Comment:
Should we supply a name and a source map @ahylton19 ?

Reply:
Yes I will make those changes